tests/track/params_test.py

```diff
@@ -1687,3 +1687,32 @@ def test_replaces_body_params(self):
         second = copy.deepcopy(search.params(choice=lambda d: d[1]))
 
         self.assertNotEqual(first, second)
+
+
+class ForceMergeParamSourceTests(TestCase):
+    def test_force_merge_index_from_track(self):
+        source = params.ForceMergeParamSouce(track.Track(name="unit-test", indices=[
+            track.Index(name="index1"),
+            track.Index(name="index2"),
+            track.Index(name="index3")
+        ]), params={})
+
+        p = source.params()
+
+        self.assertEqual("index1,index2,index3", p["index"])
+
+    def test_force_merge_index_by_name(self):
+        source = params.ForceMergeParamSouce(track.Track(name="unit-test"), params={"index": "index2"})
+
+        p = source.params()
+
+        self.assertEqual("index2", p["index"])
+
+    def test_default_force_merge_index(self):
+        source = params.ForceMergeParamSouce(track.Track(name="unit-test"), params={})
+
+        p = source.params()
+
+        self.assertEqual("_all", p["index"])
+
+
```

Comment:
`make precommit` is failing with:

```
08:13:19 ************* Module tests.track.params_test
08:13:19 tests/track/params_test.py:1718:0: C0305: Trailing newlines (trailing-newlines)
```

This is because you added two trailing lines but only one is allowed. Can you please remove this last line?

Reply:
Fixed this, and made sure make precommit passes.